src/Tizen.NUI/src/public/CustomViewRegistry.cs

```diff
@@ -98,6 +103,15 @@ public enum ScriptableType
             Default,    // Read Writable, non-animatable property, event thread only
                         //  Animatable // Animatable property, Currently disabled, UK
         }
+
+        /// <summary>
+        /// ScriptableType. Read Writable, non-animatable property, event thread only.
+        /// </summary>
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public ScriptableType Type
+        {
+            get => type;
+        }
```

Comment:
You can use expression body definition to implement a read-only property.
(https://docs.microsoft.com/en-us/dotnet/csharp/programming-guide/statements-expressions-operators/expression-bodied-members#read-only-properties)

```suggestion
        public ScriptableType Type => type;
```

Reply:
done